### DISCUSSION TITLE
result of tpv6

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#590] [Q&A] [answered]
I get this result,and I adjust the same SRs,
![image](https://user-images.githubusercontent.com/104721358/176169007-91f01ed2-1e7a-49e6-8b85-0faa7c0552cd.png)
why can not I get the same result as follow
 
![image](https://user-images.githubusercontent.com/104721358/176169388-d5d81f77-8e3c-4c16-9c2f-9b641f910253.png)

**Comment by sebwolf-de** [comment]
Hmm, your solutions seems to be really broken with spurious solutions. You need to give us more information to assess you situation. What mesh did you use?

**Reply by smallriver666** [reply]
> Hmm, your solutions seems to be really broken with spurious solutions. You need to give us more information to assess you situation. What mesh did you use?

gmsh

**Reply by daisy20170101** [reply]
Hi could you provide the following information for this issue?

**To Reproduce**
Steps to reproduce the behavior:
1. Which version do you use? Provide branch and commit id.
2. Which build settings do you use? Which compiler version do you use? 
3. On which machine does your problem occur? If on a cluster: Which modules are loaded?
4. Provide parameter/material files.

Thanks!

**Reply by smallriver666** [reply]
> Thank you for your reply,
1. 
![image](https://user-images.githubusercontent.com/104721358/176362451-120a31fc-1f0a-423f-aed9-2bbd81710396.png)

2.CC=gcc CXX=g++ FC=gfortran  CMAKE_PREFIX_PATH=~:$CMAKE_PREFIX_PATH PKG_CONFIG_PATH=~/lib/pkgconfig/:$PKG_CONFIG_PATH cmake -DNETCDF=ON -DMETIS=ON -DCOMMTHREAD=ON -DASAGI=ON -DHDF5=ON -DCMAKE_BUILD_TYPE=Release -DTESTING=OFF  -DLOG_LEVEL=warning -DLOG_LEVEL_MASTER=info -DHOST_ARCH=hsw -DPRECISION=double ..
compiler version is gcc9.3.0
3.centos7
4.parameter/material files are downloaded from
https://github.com/SeisSol/Examples

**Comment by sebwolf-de** [comment, marked as answer]
We did some work on Prakash-Clifton regularization (#596, #604). Does it work now?

**Reply by smallriver666** [reply]
I'm sorry for not replying to you in time, I've been learning some theoretical knowledge about seismic waves lately and haven't had time to log in to GitHub. I see that the source code `src` has been modified in the link you gave, does this mean that I have to recompile the software `SeisSol` and run `tpv6` again. By the way, if I want to modify this software for other calculations on this basis, what compiler do you use to debug the code, can `vscode` do it? Or any good debugging software to recommend?

**Reply by smallriver666** [reply]
> We did some work on Prakash-Clifton regularization (#596, #604). Does it work now?

ok，I recompile it, and it works now, thank you for your help.

**Comment by sebwolf-de** [comment]
1. Yes, you have to recompile SeisSol
2. gcc + gdb would be the first choice to debug.

**Reply by smallriver666** [reply]
> 1. Yes, you have to recompile SeisSol
> 2. gcc + gdb would be the first choice to debug.

ok， thank you very much.
